### DISCUSSION TITLE
How to print workflow table?

### DISCUSSION THREAD

**Discussion by alpha-code2019** [AutomaApp/automa#530] [Q&A]
Hello,
I was looking, if there is a way to export a table with **all keys and attributes**?

Thank you!

**Comment by Kholid060** [comment]
Maybe use the export data block?
![image](https://user-images.githubusercontent.com/22908993/172074834-3c94b1f1-6a89-444a-ac6b-2475c463fc16.png)

**Comment by alpha-code2019** [comment]
Tried it - but delivers "empty" *.csv

Am Mo., 6. Juni 2022 um 01:24 Uhr schrieb Ahmad Kholid <
***@***.***>:

**Reply by mzaini30** [reply]
Can you share the workflow?

**Comment by alpha-code2019** [comment]
Hello Zen,

Thank you for asking - I solved the problem by just taking screenshots of
the workflow table data

Am Mi., 8. Juni 2022 um 09:30 Uhr schrieb Zen ***@***.***>:
